map-frontend/package.json

```diff
@@ -8,14 +8,16 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "axios": "^0.19.2",
```

Comment:
Please set fixed dependency version (`"axios": "0.19.2"`)

Reply:
Fixed in f7d55ab